### PR TITLE
Ignore E402

### PR DESCRIPTION
**Pull request by ocefpaf**
Ignore errors.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +26,5 @@ script:
     #     python setup.py test';
     #   fi
     - if [[ $TEST_TARGET == 'coding' ]]; then
-        find . -type f -name "*.py" ! -name "base.py" ! -name "tests.py" | xargs flake8 --ignore=E402 --max-line-length=100;
+        find . -type f -name "*.py" ! -name "base.py" | xargs flake8 --max-line-length=100 --ignore=E402 ;
```
Comment (ocefpaf) [inline .travis.yml:29]: `tests.py` is passing now but `base.py` still has some lints. Not sure if it is worth fixing them though.

File: ODM2CZOData/NewProcessinglevelForTimeSeries.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import os
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "templatesAndSettings.settings") # noqa
-from django.core.wsgi import get_wsgi_application  # noqa
-application = get_wsgi_application()  # noqa
```
Comment (ocefpaf) [inline ODM2CZOData/NewProcessinglevelForTimeSeries.py:4]: Removed the `# noqa` as they are not needed here.

File: soilsIngestionExample/ImportMStoneData.py [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+from django.core.wsgi import get_wsgi_application
+application = get_wsgi_application()
 
+import ODM2CZOData.modelHelpers as modelHelpers
 from ODM2CZOData.models import Actions, Units, Variables
 
 __author__ = 'leonmi'
+
+
+__author__ = 'leonmi'
```
Comment (ocefpaf) [inline soilsIngestionExample/ImportMStoneData.py]: Opps. Bad rebase! Fixing this...

**Comment by ocefpaf** [comment]
@miguelcleon this one is safe to merge. It basically removes the unnecessary `noqa`.

**Comment by miguelcleon** [comment]
ok, done.

**Pull request merged**